.gitignore

```diff
@@ -1,5 +1,4 @@
 *temp*
-assets/
```

Comment:
About the move to including assets/ in the repo, and serve assets directly from the server instead of writing them out to the filesystem first and then serving them from there:

One of the use cases that I was thinking of (it was actually suggested to me by someone else) was the ability to drop testlets into the opt dir "hot" - that is, someone that is running ToDD can pop a new testlet into ```/opt/todd/server/testlets``` as an example, and since the server is scanning that directory for changes and constantly requiring agents to have those testlets else their advertisements will age out, it makes for a handy way of distributing testlets without requiring the user to recompile (or even restart) the ToDD server.

Thoughts?